src/main/resources/application.yaml

```diff
@@ -38,6 +42,19 @@ spring:
 #          lob:
 #            # silence the 'wall-of-text' - unnecessary exception throw about blob types
 #            non_contextual_creation: true
+  security:
+    oauth2:
+      client:
+        provider:
+          oidc:
+            issuer-uri: ${IDAM_OIDC_URL:http://localhost:5000}/o
+        registration:
+          oidc:
+            client-id: internal
+            client-secret: internal
+
```

Comment:
Are the following also required:
 * [idam API](https://github.com/hmcts/aac-manage-case-assignment/blob/master/src/main/resources/application.yaml#L40-L42) (i.e. if we are using `IdamClient` to get user info)
 * [S2S auth endpoint](https://github.com/hmcts/aac-manage-case-assignment/blob/master/src/main/resources/application.yaml#L55-L58) (if we are registering the `ServiceAuthFilter`)
 * [S2S authorised list](https://github.com/hmcts/aac-manage-case-assignment/blob/master/src/main/resources/application.yaml#L59-L60)